packages/example/src/App.tsx

```diff
@@ -101,45 +126,79 @@ const App = ({ examples, cells, renderers}: AppProps) => {
   return (
     <div>
       <div className='App'>
-        <header className='App-header'>
-          <img src='assets/logo.svg' className='App-logo' alt='logo' />
-          <h1 className='App-title'>Welcome to JSON Forms with React</h1>
-          <p className='App-intro'>More Forms. Less Code.</p>
+        <header className='header'>
+          <img src='assets/logo.svg' className='logo' alt='logo' />
+          <h1 className='title'>Welcome to JSON Forms with React</h1>
+          <p className='intro'>More Forms. Less Code.</p>
         </header>
         <div className='content'>
-          <h4 className='data-title'>Examples</h4>
-          <div className='data-content'>
-            <select
-              value={currentIndex}
-              onChange={ev => changeExample(Number(ev.currentTarget.value))}
-            >
-              {examples.map((optionValue: ExampleDescription, index: number) => (
-                <option
-                  value={index}
-                  label={optionValue.label}
-                  key={index}
-                >
-                  {optionValue.label}
-                </option>
-              ))}
-            </select>
+          <div className='tools'>
+            <div className='example-selector'>
+              <h4>Select Example:</h4>
+              <select
+                value={currentIndex}
+                onChange={ev => changeExample(Number(ev.currentTarget.value))}
+              >
+                {examples.map((optionValue: ExampleDescription, index: number) => (
+                  <option
+                    value={index}
+                    label={optionValue.label}
+                    key={index}
+                  >
+                    {optionValue.label}
+                  </option>
+                ))}
+              </select>
+            </div>
+            <div className='toggle-panel'>
+              <input type='checkbox' id='panel' name='panel' checked={showPanel} onChange={() => setShowPanel(!showPanel)} />
```

Comment:
Whenever a state update depends on the previous value of the state, the functional update should be used. This avoids problems with outdated variables/props.

```suggestion
              <input type='checkbox' id='panel' name='panel' checked={showPanel} onChange={() => setShowPanel(prevShow => !prevShow)} />
```